documentation/modules/security/proc-replacing-your-own-private-keys.adoc

```diff
@@ -52,6 +52,34 @@ kubectl describe Kafka _<name_of_custom_resource>_
 +
 The `type` condition changes to `ReconciliationPaused` at the `lastTransitionTime`.
 
+. Check that your `Kafka` custom resource has the `spec.clusterCa.generateCertificateAuthority` and/or the `spec.clientsCa.generateCertificateAuthority` set to `false`, depending on which CA certificate you are renewing or replacing.
+If the `generateCertificateAuthority` field is on `true` or missing, it means you want to replace the CA certificates and private keys generated by the Strimzi operators with your own.
```

Comment:
```suggestion
. Check that the `generateCertificateAuthority` properties in your `Kafka` custom resource are set `false`.
+
If the properties are set to `false`, a CA certificate is not generated by the Cluster Operator. 
You require this option is you are using your own certificates.
```

Reply:
> Check that the `generateCertificateAuthority` properties in your `Kafka` custom resource are set `false`

I was trying to make the point that not both should be set to false, it depends which CA you are replacing. It could be cluster, clients or both. With your suggestion I read as ALL of them has to be set to false.

> If the properties are set to `false`, a CA certificate is not generated by the Cluster Operator. 

Even if the field is totally missing.

Reply:
I've made a change^^